package.json

```diff
@@ -72,7 +72,7 @@
     "@ngx-formly/bootstrap": "^5.7.0",
     "@ngx-formly/core": "^5.7.0",
     "@ngx-translate/core": "^12.1.1",
-    "@rero/ng-core": "0.2.4",
+    "@rero/ng-core": "0.4",
```

Comment:
There's no `0.4` ng-core release, but `0.4.0`: https://github.com/rero/ng-core/releases

With the semantic versionning, there's always three position: major, minor, patch.

Reply:
done